app/controllers/application_controller.rb

```diff
@@ -331,28 +331,33 @@ def handle_theme
     resolve_safe_mode
     return if request.env[NO_CUSTOM]
 
-    theme_id = request[:preview_theme_id]&.to_i
+    theme_ids = []
+
+    if preview_key = request[:preview_theme_id]&.to_i
```

Comment:
why is this called preview_key? preview_theme_id ? also ... how are you going to preview user selectable theme components? something to think about.

Reply:
Weird, not sure where that came from. I'll fix it.

I can think of 2 ways for previewing user-selectable components 1) easiest is to preview CSS/HTML only with default color scheme or 2) is to allow previewing in context of a parent theme so that you can see how it looks with color schemes of different parents.